ansible_collections/arista/avd/roles/eos_cli_config_gen/docs/Input Variables.md

```diff
@@ -5321,6 +5321,270 @@ virtual_source_nat_vrfs:
+    arp_monitor_mac_address: <bool>
+    ip_proxy_arp: <bool>
+    ip_directed_broadcast: <bool>
+    ip_address: <str>
+    ip_address_secondaries:
+      - <str>
+    ip_virtual_router_addresses:
+      - <str>
+    ip_address_virtual: <str>
+    ip_address_virtual_secondaries:
+      - <str>
+    ip_igmp: <bool>
+    ip_helpers:
+      - ip_helper: <str>
+        source_interface: <str>
+        vrf: <str>
+    ipv6_enable: <bool>
+    ipv6_address: <str>
+    ipv6_address_virtual: <str>
+    ipv6_address_virtuals:
+      - <str>
+    ipv6_address_link_local: <str>
+    ipv6_virtual_router_address: <str>
+    ipv6_virtual_router_addresses:
+      - <str>
+    ipv6_nd_ra_disabled: <bool>
+    ipv6_nd_managed_config_flag: <bool>
+    ipv6_nd_prefixes:
+      - ipv6_prefix: <str>
+        valid_lifetime: <str>
+        preferred_lifetime: <str>
+        no_autoconfig_flag: <bool>
+    ip_router_virtual_address: <str>
+    access_group_in: <str>
+    access_group_out: <str>
+    ipv6_access_group_in: <str>
+    ipv6_access_group_out: <str>
+    multicast:
+      ipv4:
+        boundaries:
+          - boundary: <str>
+            out: <bool>
+        source_route_export:
+          enabled: <bool>
+          administrative_distance: <int>
+        static: <bool>
+      ipv6:
+        boundaries:
+          - boundary: <str>
+        source_route_export:
+          enabled: <bool>
+          administrative_distance: <int>
+        static: <bool>
+    ospf_network_point_to_point: <bool>
+    ospf_area: <str>
+    ospf_cost: <int>
+    ospf_authentication: <str>
+    ospf_authentication_key: <str>
+    ospf_message_digest_keys:
+      - id: <int>
+        hash_algorithm: <str>
+        key: <str>
+    pim:
+      ipv4:
+        dr_priority: <int>
+        sparse_mode: <bool>
+        local_interface: <str>
+    isis_enable: <str>
+    isis_passive: <bool>
+    isis_metric: <int>
+    isis_network_point_to_point: <bool>
+    mtu: <int>
+    no_autostate: <bool>
+    vrrp_ids:
+      - id: <int>
+        priority_level: <int>
+        advertisement:
+          interval: <int>
+        preempt:
+          enabled: <bool>
+          delay:
+            minimum: <int>
+            reload: <int>
+        timers:
+          delay:
+            reload: <int>
+        tracked_object:
+          - name: <str>
+            decrement: <int>
+            shutdown: <bool>
+        ipv4:
+          address: <str>
+          version: <int>
+        ipv6:
+          address: <str>
+    vrrp:
+      virtual_router: <str>
+      priority: <int>
+      advertisement_interval: <int>
+      preempt_delay_minimum: <int>
+      ipv4: <str>
+      ipv6: <str>
+    ip_attached_host_route_export:
+      test: <int>
+      distance: <int>
+    bfd:
+      echo: <bool>
+      interval: <int>
+      min_rx: <int>
+      multiplier: <int>
+    service_policy:
+      pbr:
+        input: <str>
+    pvlan_mapping: <str>
+    tenant: <str>
+    tags:
```

Comment:
@ClausHolbechArista  this looks like a weird output why is there no type for tags?

Reply:
good catch, let me investigate; likely an issue in the documentation generation script.

Reply:
actually was just missing some attributes to the schema:
```yaml
         tags:
          type: list
          items:
            type: str
```

Reply:
resolved in [eb7c2d3](https://github.com/aristanetworks/ansible-avd/pull/2106/commits/eb7c2d37af4d068d045f67efeb8878f565ccb12d)